salt/journal/config/srv-journal-docker-compose.yml

```diff
@@ -16,8 +16,6 @@ services:
         env_file:
             - ./containers.env
         restart: always
-        volumes:
-            - ./parameters.yml:/srv/journal/app/config/parameters.yml
```

Comment:
if I understand correctly, in YAML this `volumes` key completely override the previous `volumes` key